doc.go

```diff
@@ -941,6 +942,25 @@ Format string of the file modification time shown in the bottom line.
 
 Truncate character shown at the end when the file name does not fit to the pane.
 
+	truncatepct  int       (default 100)
+
+When a filename is too long to be shown completely, the available space is
+partitioned in two pieces. truncatepct defines a fraction (in percent
+between 0 and 100) for the size of the first piece, which will show the
+beginning of the filename. The second piece will show the end of the filename
+and will use the rest of the available space. Both pieces are separated by the
+truncation character (truncatechar).
+A value of 100 will only show the beginning of the filename,
+while a value of 0 will only show the end of the filename, e.g.:
+
+  - `set truncatepct 100` -> "very-long-filename-tr~" (default)
+
+  - `set truncatepct 50`  -> "very-long-f~-truncated"
+
+  - `set truncatepct 0`   -> "~ng-filename-truncated"
+
+.
```

Comment:
Thanks for the fix! One last nit: could you remove this period and make sure there's one blank line before the next item?